crates/runtime/src/http/routes.rs

```diff
@@ -135,7 +135,8 @@ pub(crate) fn routes(
     cors_config: &CorsConfig,
 ) -> Router {
     let mut authenticated_router = Router::new()
-        .route("/v1/sql", post(v1::query::post))
+        .route("/v1/sql", post(v1::query::sql))
+        .route("/v1/sql/prepared", post(v1::query::prepared))
```

Comment:
I wouldn't have this be a separate API - UX should be reviewed by Luke, but I'd prefer this was combined with the `/v1/sql` API and distinguished using the `Content-Type` header.